test/unit/data/schema.js

```diff
@@ -1157,6 +1157,46 @@ describe('form schema', () => {
       compare(a, b).should.be.false();
       compare(b, a).should.be.false(); // try both directions
     }));
+
+    it('should say selectMultiple matches selectMultiple', () => Promise.all([
+      fieldsFor(testData.forms.selectMultiple),
+      fieldsFor(testData.forms.selectMultiple)
+    ]).then(([ a, b ]) => {
+      compare(a, b).should.be.true();
+      compare(b, a).should.be.true(); // try both directions
+    }));
+
+    it('should say select1 and selectMultiple are different', () => {
+      const selectOne = `<?xml version="1.0"?>
+      <h:html xmlns="http://www.w3.org/2002/xforms" xmlns:h="http://www.w3.org/1999/xhtml" xmlns:ev="http://www.w3.org/2001/xml-events" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:jr="http://openrosa.org/javarosa">
+        <h:head>
+          <model>
+            <instance>
+              <data id="selectMultiple">
+                <q1/>
+                <g1><q2/></g1>
+              </data>
+            </instance>
+            <bind nodeset="/data/q1" type="string"/>
+            <bind nodeset="/data/g1/q2" type="string"/>
+          </model>
+        </h:head>
+        <h:body>
+          <select1 ref="/data/q1"><label>one</label></select1>
+          <group ref="/data/g1">
+            <label>group</label>
+            <select1 ref="/data/g1/q2"><label>two</label></select1>
+          </group>
+        </h:body>
+      </h:html>`;
```

Comment:
Is this the same XML as what's in the integration test? Is it worth including it in `testData.forms`? (Maybe nearby `testData.forms.selectMultiple` in case that form ever changes?)

Reply:
I thought about this, and even though it's used twice, I'm not sure it should go there. One main reason is the id is still called 'selectMultiple'. Maybe once there's a >3x usage of the form or something!